### DISCUSSION TITLE
添加包的时候如何禁用某一个plat下面的arch

### DISCUSSION THREAD

**Discussion by da-liii** [xmake-io/xmake#4267] [General]
比如CI除了Windows的arm64架构，全部通过，想临时禁用arm64。

**Comment by da-liii** [comment]
https://xmake.io/#/zh-cn/manual/package_dependencies?id=%e5%b9%b3%e5%8f%b0%e8%bf%87%e6%bb%a4

**Comment by star-hengxing** [comment]
好像不能偷懒，一定要写满平台
